src/page/template/content/connect-requests.htm

```diff
@@ -4,8 +4,8 @@
       <div class="connect-request" data-bind="attr: {'data-uie-uid': id}" data-uie-name="connect-request">
         <div class="connect-request-name ellipsis" data-bind="text: name()"></div>
         <div class="connect-request-username label-username" data-bind="text: handle"></div>
-        <!-- ko if: classifiedDomains -->
-          <classified-bar params="users: [$data], classifiedDomains"></classified-bar>
+        <!-- ko if: $parent.classifiedDomains -->
```

Comment:
Since we are in a foreach loop with isolated syntax, we need to manually access the parent viewmodel